### ISSUE TITLE
Update to use OpenAPI Generator?

### ISSUE THREAD

**Issue by lauracowen** [OpenLiberty/guide-microprofile-rest-client#114]
Should this guide be updated to use the OpenAPI Generator to more quickly create the REST client code?

**Comment by yeekangc** [comment]
As in the one we created for VS Code?  Or you meant Open API Tools in general?

**Comment by yeekangc** [comment]
IMO, it will be too much to cover the Open API Tools and the VS Code generator in the guide itself. We can mention and reference them as tools that users can consider and use.

Covering them may distract users from the learning objectives, which is around MP Rest Client, and introducing additional dependencies too.

@gkwan-ibm @NimG98, please consider.

FYI, @turkeylurkey @ericglau @kathrynkodama.

**Comment by lauracowen** [comment]
Yes, I did wonder if it would distract from the material itself. That sounds fair.
